modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb

```diff
@@ -104,5 +104,9 @@ def service(auth_headers)
         EVSS::DisabilityCompensationForm::Service.new(auth_headers)
       end
     end
+
+    def get_errors(error)
+      error&.errors if error.methods.include?(:errors)
+    end
```

Comment:
this returns `nil` if it doesn't hit the `if`... which would mean `error&.errors` would also return `nil` - do we need this method added with the safe navigator?